features/shared/checkout_previous_branch_after_success/deleted_current_branch/previous_branch_same.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Allow checking out the correct previous Git branch after running a Git Town commmand that keeps the previous branch intact and deletes the current branch
```

Comment:
Feature: Git Town commands that delete the current branch keep the previous Git branch as-is